### DISCUSSION TITLE
The following error occurs when the dpgen task is submitted on the supercomputer

### DISCUSSION THREAD

**Discussion by 12jscvb** [deepmodeling/dpgen#1400] [Q&A]
![QQ图片20231111172241](https://github.com/deepmodeling/dpgen/assets/145745956/e960bff3-8ec6-4eaa-8541-1f653137c91d)
This error message indicates that your batch job failed to submit，I don't know how to solve it，Thank you for help

**Comment by njzjz** [comment]
Please check Slurm documentation https://slurm.schedmd.com/resource_limits.html
